package.json

```diff
@@ -3,70 +3,32 @@
   "name": "davidia",
   "version": "0.1.0",
   "private": true,
-  "type": "module",
+  "packageManager": "pnpm@8.6.0",
   "scripts": {
-    "preinstall": "npx only-allow pnpm",
-    "start": "vite",
+    "start": "pnpm --filter {client/example} start",
     "build": "vite build",
-    "preview": "vite preview",
+    "packages": "pnpm --filter {client/*} build",
+    "start:storybook": "pnpm --filter storybook start",
+    "build:storybook": "pnpm --filter storybook build",
     "test": "vitest",
     "prettify": "prettier -w client/**",
     "lint": "eslint \"client/**\" --ext .ts --ext .tsx",
```

Comment:
Can simplify these scripts commands as you have set the ignore patterns. Also rationalize so these top level scripts call the scripts in lower level package.jsons